packages/l10n/scripts/generate-l10n-data.js

```diff
@@ -294,6 +296,114 @@ const mapDiffToWarnings = (oldData, newData) => {
     .filter(Boolean);
 };
 
+const updateTimeZoneData = async (key) => {
+  const allTimeZoneIds = moment.tz.names();
+  // uncomment for testing/review purposes, will be removed once PR is out of draft
+  // const fakeTimeZones = ['zzspace/moon', 'zzspace/mars', 'zzspace/jupiter'];
+  // const allTimeZoneIds = moment.tz.names().concat(fakeTimeZones);
+  const dataFolderPath = path.join(__dirname, '..', DATA_DIR[key].path);
+  const sourceDataFilePath = path.join(dataFolderPath, 'core.json');
+  await fs.accessSync(sourceDataFilePath, fs.constants.F_OK);
```

Comment:
awaiting something sync is odd to me. Maybe we should extract it into `function exitIfSourceFolderDoesNotExist` as that's what we want I think.

Reply:
I don't think this actually needs an `await`, I'll remove it.